io.github.prateekmedia.appimagepool.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+rename-icon: appimagepool
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: appimagepool
+    buildsystem: simple
+    only-arches:
+      - x86_64
+    build-commands:
+      - install -Dm644 appimagepool/appimagepool /app/appimagepool/appimagepool
+      - cp -R appimagepool/data /app/appimagepool/data
+      - cp -R appimagepool/lib /app/appimagepool/lib
+      - chmod +x /app/appimagepool/appimagepool
+      - chmod +x /app/appimagepool
+      - install -Dm644 appimagepool/appimagepool.svg /app/share/icons/hicolor/scalable/apps/appimagepool.svg
+      - mkdir /app/bin
+      - mkdir -p /app/lib/
+      - ln -s /usr/lib/x86_64-linux-gnu/libgio-2.0.so.0 /app/lib/libgio-2.0.so
+      - if [ ! -e '/app/lib/libgio-2.0.so' ]; then ln -s -f /usr/lib/aarch64-linux-gnu/libgio-2.0.so.0 /app/lib/libgio-2.0.so; fi
```

Comment:
given you don't have a aarch64 I wonder what you are trying to achieve. Also I'm not sure why you need to symlink `libgio-2.0.so`

Reply:
I will use it for future version as I will add support for it in near future.

The app does supports aarch64 but I am not building for it due to docker not being setup.

Reply:
Also I need fo symlink libgio cause without it app doesnt open and complains about libgio symlink, try this for demo  https://github.com/flathub/flathub/pull/2492#issuecomment-913009342